src/patternfly/demos/PasswordStrength/examples/PasswordStrength.md

```diff
@@ -0,0 +1,175 @@
+---
+id: 'Password strength'
+beta: true
+section: demos
+---
+
+## Examples
+
+### Initial state
+```hbs
+  {{#> form}}
+    {{#> form-group}}
+      {{#> split split--modifier="pf-m-gutter"}}
+        {{#> split-item split-item--modifier="pf-m-fill"}}
+          {{#> form-label form-label--attribute='for="invalid-login-demo-form-password"' required="true"}}Password{{/form-label}}
```

Comment:
One thing we do a lot is give the `{{#> form}}` block a `form--id="helper-text-invalid"`, then do `form-label--attribute=(concat 'for="' form--id '-password"')`, so we can use that anywhere we need a unique string that's used multiple places in an example, then use that same code in each example with a unique `form--id`.